kernel/src/main/java/org/kframework/compile/ExpandMacros.java

```diff
@@ -107,6 +107,19 @@ public ExpandMacros(ResolveFunctionWithConfig transformer, Module mod, FileUtil
         }
     }
 
+    private int getPriority(Att att) {
+        if (att.contains(Att.PRIORITY())) {
+            try {
+                return Integer.parseInt(att.get(Att.PRIORITY()));
+            } catch (NumberFormatException e) {
+                throw KEMException.compilerError("Invalid value for priority attribute: " + att.get(Att.PRIORITY()) + ". Must be an integer.", e);
+            }
+        } else if (att.contains(Att.OWISE())) {
+            return 200;
+        }
+        return 50;
+    }
```

Comment:
This same function should be re-used for `ModuleToKORE` to make sure that we keep any logic about how to compute the priority of a rule in sync.